webapp/src/views/ProfileDashboard/Organizations/Organizations.js

```diff
@@ -48,48 +48,25 @@ function Organizations() {
 	useEffect(
 		() => {
 			// Make sure we have a value (user has entered something in input)
-			// TODO - refactor - copy-paste code in both if-else blocks
-			if (debouncedSearchTerm) {

```

Comment:
Deleting this will break the search functionality. The search uses a debounce hook to perform a search request when the user has stopped typing for 0,5s.